src/Extension/HeadingPermalink/HeadingPermalinkExtension.php

```diff
@@ -31,6 +31,8 @@ public function configureSchema(ConfigurationBuilderInterface $builder): void
             'max_heading_level' => Expect::int()->min(1)->max(6)->default(6),
             'insert' => Expect::anyOf(HeadingPermalinkProcessor::INSERT_BEFORE, HeadingPermalinkProcessor::INSERT_AFTER)->default(HeadingPermalinkProcessor::INSERT_BEFORE),
             'id_prefix' => Expect::string()->default('content'),
+            'attach_heading' => Expect::bool()->default(false),
+            'heading_class' => Expect::string()->default('heading-anchor'),
```

Comment:
I'd strongly prefer that this defaults to an empty string since there's a small possibility that users might have existing CSS styles targeting `.heading-anchor` for other purposes and so upgrading `league/commonmark` to 2.4 may unexpectedly break site styles.

Reply:
Up to you, I will empty it